sensing/pointcloud_preprocessor/docs/blockage_diag.md

```diff
@@ -58,14 +58,16 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 | `horizontal_ring_id`          | int    | The id of horizontal ring of the LiDAR                                                                                        |
```

Comment:
[nits] this param is not described in parameter file?

Reply:
@badai-nguyen
sorry now I understand `horizontal_ring_id` is defined in launcher file.
If there is some reasonable reason for keeping this implementation, no problem for me.
But if no time to move to parameter file and remove defaulat value, I will do that for my learning.

Reply:
@kyoichi-sugahara Sorry for late response. As my understanding,not only `blockage_diag`, some other parameters of `pointcloud_preprocessor`  also are defined in launcher file since they are specific for each sensor configuration. 
I think we can move to parameter file by separate the common parameters file and parameter specific for each sensor file.

Reply:
@badai-nguyen 
Thank you so much for the answer!
I understand and resolve conversation :heart_eyes: